instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -584,20 +636,31 @@ def __call__(self, environ, start_response):
         try:
             with trace.use_span(span):
                 start_response = self._create_start_response(
-                    span, start_response, response_hook, duration_attrs
+                    span,
+                    start_response,
+                    response_hook,
+                    req_attrs,
+                    self._sem_conv_opt_in_mode,
                 )
                 iterable = self.wsgi(environ, start_response)
                 return _end_span_after_iterating(iterable, span, token)
         except Exception as ex:
             if span.is_recording():
+                if _report_new(self._sem_conv_opt_in_mode):
+                    span.set_attribute(_SPAN_ATTRIBUTES_ERROR_TYPE, type(ex).__qualname__ )
                 span.set_status(Status(StatusCode.ERROR, str(ex)))
             span.end()
             if token is not None:
                 context.detach(token)
             raise
         finally:
-            duration = max(round((default_timer() - start) * 1000), 0)
-            self.duration_histogram.record(duration, duration_attrs)
+            duration_s = default_timer() - start
+            if self.duration_histogram_old:
+                duration_attrs_old = _parse_duration_attrs(req_attrs, _HTTPStabilityMode.DEFAULT)
+                self.duration_histogram_old.record(max(round(duration_s * 1000), 0), duration_attrs_old)
+            if self.duration_histogram_new:
+                duration_attrs_new = _parse_duration_attrs(req_attrs, _HTTPStabilityMode.HTTP)
+                self.duration_histogram_new.record(max(round(duration_s * 1000), 0), duration_attrs_new)
```

Comment:
the new one should be in seconds
```suggestion
                self.duration_histogram_new.record(max(round(duration_s), 0), duration_attrs_new)
```